index.html

```diff
@@ -225,6 +225,57 @@ <h4>Install the client on your headset</h4>
                             </div>
                         </div>
 
+                        <div class="row gx-0 my-5 justify-content-center">
+                            <h4>Settings guide</h4>
+
+                            <ul class="nav nav-tabs" id="myTab2" role="tablist">
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link active" id="foveation-tab" data-bs-toggle="tab" data-bs-target="#foveation" type="button" role="tab" aria-controls="adb" aria-selected="true">Foveation</button>
+                                </li>
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link" id="bitrate-tab" data-bs-toggle="tab" data-bs-target="#bitrate" type="button" role="tab" aria-controls="bitrate" aria-selected="true">Bitrate</button>
+                                </li>
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link" id="enconders-tab" data-bs-toggle="tab" data-bs-target="#encoders" type="button" role="tab" aria-controls="encoders" aria-selected="false">Encoders</button>
+                                </li>
+                            </ul>
+                            <div class="tab-content" id="myTabContent2">
+                                <div class="tab-pane fade show active" id="foveation" role="tabpanel" aria-labelledby="foveation-tab">
+                                    <p>A stronger foveation makes the image sharper in the center than in the periphery and makes the decoding faster. This is better for fast paced games. </p>
+                                    <p>A weaker foveation gives a uniform sharpness in the whole image.</p>          
+                                    <p>The recommended values are between 20% and 50% for headsets without eye tracking and between 50% and 70% for headsets with eye tracking.</p>
+                                    <img src="screenshots/settings/foveation.jpg" class="d-block w-100">
```

Comment:
The screenshot repeats a lot from the paragraph.

Have you checked if it would look better with a single screenshot of the interface for all tabs?

Also I would use png for UI screenshots, it prevents compression artefacts